.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "githubPullRequests.ignoredPullRequestBranches": [
+        "master"
+    ]
+}
```

Comment:
what is this

Reply:
oh, i was using github.dev and it gave me this error message, lemme remove that